core/editor/src/frontend/frontend_message_handler.rs

```diff
@@ -12,14 +12,14 @@ pub enum FrontendMessage {
 	ExpandFolder { path: Vec<LayerId>, children: Vec<LayerPanelEntry> },
 	SetActiveTool { tool_name: String },
 	SetActiveDocument { document_index: usize },
-	CloseDocument { document_index: usize },
-	NewDocument { document_name: String },
+	UpdateOpenDocumentsList { open_documents: Vec<String> },
+	PromptConfirmationToCloseDocument { document_index: usize },
```

Comment:
I believe "confirm" and "prompt" are two different terms for types of dialogs, at least in web dev. Consider something like `ConfirmCloseDocument`. I think that would be less verbose and equally/more clear, especially if/when more `confirm`s are added.

Reply:
I agree, that would be good.